packages/react-components/react-combobox/src/components/Combobox/useCombobox.tsx

```diff
@@ -222,6 +226,29 @@ export const useCombobox_unstable = (props: ComboboxProps, ref: React.Ref<HTMLIn
   if (state.expandIcon) {
     state.expandIcon.onMouseDown = onExpandIconMouseDown;
     state.expandIcon.onClick = onExpandIconClick;
+
+    // If there is no explicit aria-label, calculate default accName attribute for expandIcon button,
+    // using the following steps:
+    // 1. If there is an aria-label, it is "Open [aria-label]"
+    // 2. If there is an aria-labelledby, it is "Open [aria-labelledby target]" (using aria-labelledby + ids)
+    // 3. If there is no aria-label/ledby attr, it falls back to "Open"
+    // We can't fall back to a label/htmlFor name because of https://github.com/w3c/accname/issues/179
+    const hasExpandLabel = state.expandIcon['aria-label'] || state.expandIcon['aria-labelledby'];
+    const defaultOpenString = 'Open'; // this is english-only since it is the fallback
```

Comment:
I know we don't have our translation support fully baked yet so can you add an issue to follow up on this once we get translations sorted? It would be good to allow this fallback to be properly translated in the future.

Reply:
issue here: https://github.com/microsoft/fluentui/issues/26287